### PR TITLE
feat: Added Meshery Operator Blog

### LINKED ISSUE #3327: [Blog]: Meshery Operator
Contributors and community members are encouraged to post on https://layer5.io/blog. A new blog post fitting the following description is requested to be written by a community member:

**Desired Title:** What is Meshery Operator?

**Topic:** Meshery Operator

**Core points to convey in the post:** 
1. General description of Kubernetes Operators
2. Description of each of Meshery Operator's custom controllers and their function (include MeshSync logo)

---
<h3>Contributor Resources</h3>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).

### PR DESCRIPTION
**Pull request by thisiskaransgit**
**Description**

This PR fixes #3327 
- Added a blog about Kubernetes Operators and what is Meshery operators and the two controllers that it manages.

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Review by Aisuko — COMMENTED** [review]
cc @leecalcote

**Comment by leecalcote** [comment]
@Aisuko look good to you? Did we miss anything?

File: src/collections/blog/2023/2023-06-21-meshery-operators/index.mdx [hunk 1]
```diff
@@ -0,0 +1,84 @@
+---
+title: "What is Meshery Operator"
+subtitle: "Describing Meshery operator, what it is and how it simplifies management"
+date: 2023-06-21 10:30:05 -0530
+author: Karan Thakur
+thumbnail: ./gsoc-wide.webp
+darkthumbnail: ./gsoc-wide.webp
```
Comment (leecalcote) [inline src/collections/blog/2023/2023-06-21-meshery-operators/index.mdx:7]: Let's get another image swapped in here...

File: src/collections/blog/2023/2023-06-21-meshery-operators/index.mdx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+category: "Meshery"
+description: "Streamlining Cloud native Management with Meshery Operators and it's Custom Controllers"
```
Comment (leecalcote) [inline src/collections/blog/2023/2023-06-21-meshery-operators/index.mdx:9]:
```suggestion
description: "Streamlining Cloud Native Management"
```

**Comment by Aisuko** [comment]
> @Aisuko look good to you? Did we miss anything?

Thanks @leecalcote, It is good for me.

**Comment by l5io** [comment]
🚀 Preview for commit 609efcaf3cb4cfe64fc854cb25e80304caddd388 at: https://64a55d9baa30b604a40edfca--layer5.netlify.app

File: src/collections/blog/2023/2023-06-21-meshery-operators/index.mdx (hunk 1 above)
Comment (thisiskaransgit) [reply src/collections/blog/2023/2023-06-21-meshery-operators/index.mdx]: Done

**Comment by thisiskaransgit** [comment]
@leecalcote @Abhishek-kumar09 
- Updated Thumbnail
- Added CTA, at the conclusion 
- Added Meshsync logo

File: src/collections/blog/2023/2023-07-05-meshery-operators/index.mdx [hunk 3]
```diff
@@ -0,0 +1,93 @@
+---
+title: "What is Meshery Operator"
+subtitle: "Describing Meshery operator, what it is and how it simplifies management"
+date: 2023-07-05 10:30:05 -0530
+author: Karan Thakur
```
Comment (thisiskaransgit) [inline src/collections/blog/2023/2023-07-05-meshery-operators/index.mdx:5]:
```suggestion
author: Layer5 team
```
If it passes the quality check

**Comment by l5io** [comment]
🚀 Preview for commit 3ab0752fbc0733d23cec0dcbbfc7741b798fba9e at: https://64a59412a7cf7603de114cb4--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit cf42a6805456cd51b048fe5ce7e3d48d4a1b8c02 at: https://64a7128a2be2da6d35f6d416--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 11071aa098ed9c2b39d29bd502a0862272038b15 at: https://64b85321cbdc5500840c8a0b--layer5.netlify.app

**Review by Chadha93 — CHANGES_REQUESTED** [review]
Looks good @thisiskaransgit, just a quick fix to the padding of operator image required. 
image

**Review by Abhishek-kumar09 — APPROVED** [review]
looking concise and well written.

**Comment by Abhishek-kumar09** [comment]
@Chadha93 If padding is the only blocker, then lets not wait. Apply the changes and merge it with the updated date.

File: src/collections/blog/2023/2023-07-05-meshery-operators/index.mdx [hunk 4]
```diff
@@ -0,0 +1,93 @@
+---
+title: "What is Meshery Operator"
+subtitle: "Describing Meshery operator, what it is and how it simplifies management"
+date: 2023-07-05 10:30:05 -0530
```
Comment (Chadha93) [inline src/collections/blog/2023/2023-07-05-meshery-operators/index.mdx:4]:
```suggestion
date: 2023-07-30 10:30:05 -0530
```

**Review by Chadha93 — APPROVED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit 17e4d479352f61f5d18a44d356b479349253e77c at: https://64c685704de7b11372518df9--layer5.netlify.app

File: src/collections/blog/2023/2023-07-05-meshery-operators/index.mdx (hunk 3 above)
Comment (leecalcote) [reply src/collections/blog/2023/2023-07-05-meshery-operators/index.mdx:5]: Missing capitalization of Operator

File: src/collections/blog/2023/2023-07-05-meshery-operators/index.mdx [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+### Use cases 
+
+1. Kubernetes Operators provide an advantage by extending the functionality of Kubernetes to stateful applications, in addition to stateless ones. This is important because managing stateful cloud applications and services can be more complex than managing stateless ones.
+
+2. The use of Operators helps standardize manual activities, creating a consistent approach to automation.
+
+3. One benefit of using Operators is their ease of transportability between environments and projects. This has led to the development of an ecosystem with many generic Operators that can be downloaded, configured, and used across different projects without needing in-house development efforts. 
+
+Meshery Operator is one such operator built specifically for Meshery cloud native management platform. It provides a declarative way to install, configure and maintain Meshery instances running on Kubernetes platforms while also providing better workflows which simplify much of the complexity involved in deploying/maintaining these systems.
```
Comment (leecalcote) [inline src/collections/blog/2023/2023-07-05-meshery-operators/index.mdx:41]: The lead-in before mentioning Meshery Operator is too long.

File: src/collections/blog/2023/2023-07-05-meshery-operators/index.mdx [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+
+Meshery Operator is a multi-service mesh operator (a Kubernetes custom controller) created by Layer5 community to simplify the lifecycle management of Meshery instances.
+<figure class="imgWithCaption fig-right" style="width:30%" >
+    <img src={operator}  />
+    <figcaption>
+    <p>Meshery Operator</p>
+    </figcaption>
+ </figure>
+
+Meshery Operator automates the installation and lifecycle management of Meshery service mesh management platform. It uses custom controllers to manage different aspects of Meshery deployment, including configuration, scaling, monitoring, and maintenance.
```
Comment (leecalcote) [inline src/collections/blog/2023/2023-07-05-meshery-operators/index.mdx:56]: No more service mesh management

**Pull request merged**